src/execution/__tests__/lists-test.ts

```diff
@@ -113,7 +113,7 @@ describe('Execute: Accepts async iterables as list value', () => {
         },
       }),
     });
-    return execute({
+    return experimentalExecuteIncrementally({
```

Comment:
The tests in this file don't use defer or stream. Why did you change it to use `experimentalExecuteIncrementally`?

Reply:
Good catch. It doesn't need to be, and my PR will now remove the `| AsyncGenerator` from `completeObjectList`'s return type, making it even more clear that the tests in this file don't use defer or stream.